app/migrations/Version20230927055621.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+
+final class Version20230927055621 extends AbstractMauticMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table = $schema->getTable($this->prefix.'emails');
+
+        if (!$table->hasColumn('continue_sending')) {
+            $this->addSql("ALTER TABLE {$this->prefix}emails ADD `continue_sending` TINYINT(1) DEFAULT 0");
```

Comment:
The not null must be specified, right? It's not implied.
```suggestion
            $this->addSql("ALTER TABLE {$this->prefix}emails ADD `continue_sending` TINYINT(1) DEFAULT 0 NOT NULL");
```